demos-europe/demosplan-core
# bug(refs T30439): add controls to map with masterportal

**Ticket:** https://yaits.demos-deutschland.de/T30439

The controls were missing after implementing the masterportal api for creating the map in DpOlMap.

### How to review/test
Go to Verfahren -> Grundeinstellungen -> Verortung des Verfahrens and see if the buttons for fullscreen and scrolling are shown in the map and working.
